azure-pipelines.yml

```diff
@@ -6,56 +6,33 @@ trigger:
     include:
     - '*'
 jobs:
-- job: Tox36
-  displayName: 'Tox: Python 3.6'
+- job: Tox
   condition: succeeded()
   pool:
-    vmImage: 'ubuntu-16.04'
-  steps:
-  - task: UsePythonVersion@0
-    displayName: 'Use Python 3.6'
-    inputs:
-      versionSpec: 3.6
-  - task: Bash@3
-    displayName: 'Run Tox'
-    env:
-      TOXENV: py36
-    inputs:
-      targetType: 'filePath'
-      filePath: scripts/ci/run_tox.sh
-
-- job: Tox37
-  displayName: 'Tox: Python 3.7'
-  condition: succeeded()
-  pool:
-    vmImage: 'ubuntu-16.04'
-  steps:
-  - task: UsePythonVersion@0
-    displayName: 'Use Python 3.7'
-    inputs:
-      versionSpec: 3.7
-  - task: Bash@3
-    displayName: 'Run Tox'
-    env:
-      TOXENV: py37
-    inputs:
-      targetType: 'filePath'
-      filePath: scripts/ci/run_tox.sh
-
-- job: Tox38
-  displayName: 'Tox: Python 3.8'
-  condition: succeeded()
-  pool:
-    vmImage: 'ubuntu-16.04'
+    vmImage: 'ubuntu-18.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+        tox_env: 'py36'
+      Python37:
+        python.version: '3.7'
+        tox_env: 'py37'
+      Python38:
+        python.version: '3.8'
+        tox_env: 'py38'
+      Python39:
+        python.version: '3.9'
+        tox_env: 'py39'
```

Comment:
Using [matrix](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#matrix) to avoid duplicated code.